website/docs/deployment.md

```diff
@@ -21,7 +21,7 @@ These commands are equivalent:
 ```
 
 ```shell
-(venv) $ uvicorn foo:main
+(venv) $ H2O_WAVE_APP_ADDRESS="http://127.0.0.1:8000" uvicorn foo:main
```

Comment:
It would be better to explain why this might be needed. See https://github.com/h2oai/wave/discussions/1803#discussioncomment-4757071